### PR TITLE
blackoilextbomodules: put parameters in separate class

### PR DESCRIPTION
**Pull request by akva2**
limits the amount of static variable declarations

### CODE REVIEW

**Comment by akva2** [comment]
jenkins build this please

File: opm/models/blackoil/blackoilextboparams.hh [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+#define EWOMS_BLACK_OIL_EXTBO_PARAMS_HH
+
+#include <opm/material/common/Tabulated1DFunction.hpp>
+#include <opm/material/common/UniformXTabulated2DFunction.hpp>
+
+#include <vector>
+
+namespace Opm {
+
+template<class Scalar>
```
Comment (blattms) [inline opm/models/blackoil/blackoilextboparams.hh:42]: Some doxygen documentation at least about the class might be handy for fellow developers.

**Review by blattms — APPROVED** [review]
Looks good, would prefer some documentiation for the new class though.

File: opm/models/blackoil/blackoilextboparams.hh (hunk 1 above)
Comment (akva2) [reply opm/models/blackoil/blackoilextboparams.hh:42]: i totally agree. problem is i'm a bit in deep water as to what the various parameters are for :/ but at least i can doxy what the class is used for.

**Comment by akva2** [comment]
jenkins build this please

**Pull request merged**